content/source/docs/cloud/api/state-versions.html.md

```diff
@@ -163,96 +163,180 @@ curl \
 {
     "data": [
         {
-            "id": "sv-SDboVZC8TCxXEneJ",
+            "id": "sv-g4rqST72reoHMM5a",
             "type": "state-versions",
             "attributes": {
-                "vcs-commit-sha": null,
-                "vcs-commit-url": null,
-                "created-at": "2018-08-27T14:49:47.902Z",
+                "created-at": "2021-06-08T01:22:03.794Z",
+                "size": 940,
                 "hosted-state-download-url": "https://archivist.terraform.io/v1/object/...",
-                "serial": 3
+                "modules": {
+                    "root": {
+                        "null-resource": 1,
+                        "data.terraform-remote-state": 1
+                    }
+                },
+                "providers": {
+                    "provider[\"terraform.io/builtin/terraform\"]": {
+                        "data.terraform-remote-state": 1
+                    },
+                    "provider[\"registry.terraform.io/hashicorp/null\"]": {
+                        "null-resource": 1
+                    }
+                },
+                "resources": [
+                    {
+                        "name": "other_username",
+                        "type": "data.terraform_remote_state",
+                        "count": 1,
+                        "module": "root",
+                        "provider": "provider[\"terraform.io/builtin/terraform\"]"
+                    },
+                    {
+                        "name": "random",
+                        "type": "null_resource",
+                        "count": 1,
+                        "module": "root",
+                        "provider": "provider[\"registry.terraform.io/hashicorp/null\"]"
+                    }
+                ],
+                "resources-processed": false,
+                "serial": 9,
+                "state-version": 4,
+                "terraform-version": "0.15.4",
+                "vcs-commit-url": "https://gitlab.com/nfagerlund/terraform-minimum/-/commit/5cffb47f35a2ab2ab8faf8f07b030c9099775a6f",
```

Comment:
I'm not sure if you meant to include this URL + the VCS URLs in the other responses - would it be better to replace those with anonymized/dummy data?

```suggestion
                "vcs-commit-url": "https://gitlab.com/my-organization/terraform-test/-/commit/abcdef12345",
```

Reply:
I didn't, thank you for catching that! Caught a bunch of the other identifiers, but forgot that one.